web/src/i18n/locales/en/apd/stateProfile.yaml

```diff
@@ -43,7 +43,7 @@ keyPersonnel:
     email: Email
     percentTime: Percent commitment to program
     primary: 'This person is the primary APD point of contact:'
-    hasCosts: 'This person has costs directly attributable to the program:'
+    hasCosts: Does this person have costs directly attributable to this project?
```

Comment:
@jameshupp 

![shhh](https://media.giphy.com/media/TXRHoV5Dvv7cA/giphy.gif)

Reply:
This should say `program` not `project` to maintain consistency. [This exact discrepancy came up a little while back](https://github.com/18F/cms-hitech-apd/issues/1145#issuecomment-471100068); both the activities and the personnel costs support a _program_. The antecedent of _project_ isn't really specified in the app.

But phrasing it as a question is great!